### PR TITLE
UINV-65 show error message

### PR DESCRIPTION
**Pull request by Feboll**
## Purpose

https://issues.folio.org/browse/UINV-65

## Approach

Show error message

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceDetails/InvoiceActions/ApproveInvoiceAction/ApproveInvoiceAction.js [hunk 1]
```diff
@@ -24,7 +25,17 @@ const ApproveInvoiceAction = ({ saveInvoice, invoice }) => {
...
 
       saveInvoice(approvedInvoice)
-        .then(() => showCallout('ui-invoice.invoice.actions.approve.success'));
+        .then(() => showCallout('ui-invoice.invoice.actions.approve.success'))
+        .catch(async (response) => {
+          try {
+            const { errors } = await response.json();
+            const errorCode = get(errors, [0, 'code']);
+
+            showCallout(`ui-invoice.invoice.actions.approve.error.${errorCode}`, 'error');
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceDetails/InvoiceActions/ApproveInvoiceAction/ApproveInvoiceAction.js]: what if it's a new code and there is no translation on ui yet?

Comment (Feboll) [reply src/invoices/InvoiceDetails/InvoiceActions/ApproveInvoiceAction/ApproveInvoiceAction.js]: We will get toast with code for translation. 
May be I can use message from this error? Or I need every time display default message?

Comment (aliaksei-chumakou) [reply src/invoices/InvoiceDetails/InvoiceActions/ApproveInvoiceAction/ApproveInvoiceAction.js]: don't you think that showing to user some translation keys isn't the best idea? 
it's also a case when new code introduced on the backend side 
I think you need constants for specific error codes you work in scope of this ticket, and for the rest to show general message

Comment (Feboll) [reply src/invoices/InvoiceDetails/InvoiceActions/ApproveInvoiceAction/ApproveInvoiceAction.js]: You are right. I updated with constants.

File: src/common/utils/getApproveErrorMessage.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const APPROVE_ERROR_CODES = {
+  voucherNumberPrefixNotAlpha: 'voucherNumberPrefixNotAlpha',
+  FundDistributionsNotPresent: 'FundDistributionsNotPresent',
+};
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (AndreiShumski2) [inline src/common/utils/getApproveErrorMessage.js]: no needed this line

Comment (Feboll) [reply src/common/utils/getApproveErrorMessage.js]: fixed

**Pull request merged**